src/mainboard/clevo/adl-p/Kconfig

```diff
@@ -4,7 +4,7 @@ config BOARD_CLEVO_ADLP_COMMON
 	select DRIVERS_I2C_HID
 	select DRIVERS_INTEL_PMC
 	select DRIVERS_INTEL_USB4_RETIMER
-	select EC_CLEVO_EC
+	select EC_SYSTEM76_EC
```

Comment:
Maybe for upstream EC_CLEVO_EC should be selected here by default, and be configurable (also changed in NovaCustom directory)?

Reply:
let's go back to it